posthog/celery.py

```diff
@@ -192,6 +195,42 @@ def pg_table_cache_hit_rate():
             pass
 
 
+@app.task(ignore_result=True)
+def pg_plugin_server_query_timing():
+    from posthog.internal_metrics import gauge
+
+    with connection.cursor() as cursor:
+        try:
+            cursor.execute(
+                """
+                SELECT
+                    substring(query from 'plugin-server:(\\w+)') AS query_type,
```

Comment:
Tiny nit: this would be nicer to copy and paste if this were a raw string literal (`r"""`), removing the need for escaping the backslash.

Reply:
Tested: I would still need to escape %%plugin-server%% below so switching to raw would only eliminate the escaping on this line. We also don't really use raw strings for queries elsewhere.